### DISCUSSION TITLE
Sprint-1 Done

### DISCUSSION THREAD

**Discussion by Officer-R** [gatsbyjs/gatsby#29126] [Community]
Hey guys hope you all are doing well!

This is my first time experience with gatsby. I tried to implement all the requirements but I am not sure whether i should source data from Contentful. As of now I did not source data from any headless CMS.

Also did not use file system route api.

Anyways here's the link: [AudioC0re](https://audioc0resprint1.gtsb.io)

Regards 
Rohit.S

**Comment by Aereli** [comment]
I used Sanity.io as my headless CMS as I've had prior experience to it. Do you have a Repo to your site? This one is my [project](https://100daysofgatsby2021.netlify.app/) incase anyone else would like to see. Also I deployed on netlify.

**Reply by Officer-R** [reply]
Yes I have a repo to my site.

[Repo link](https://github.com/rohit1101/AudioC0RE)

Check out the backup branch.
